src/Workspaces/Core/Portable/FindSymbols/Declarations/DeclarationFinder_AllDeclarations.cs

```diff
@@ -93,18 +93,20 @@ internal static partial class DeclarationFinder
         private static async Task<(bool, ImmutableArray<SymbolAndProjectId>)> TryFindAllDeclarationsWithNormalQueryInRemoteProcessAsync(
             Project project, SearchQuery query, SymbolFilter criteria, CancellationToken cancellationToken)
         {
-            var session = await SymbolFinder.TryGetRemoteSessionAsync(
-                project.Solution, cancellationToken).ConfigureAwait(false);
-            if (session != null)
+            using (var session = await SymbolFinder.TryGetRemoteSessionAsync(
```

Comment:
this one we want to hold onto session otherwise, service in the server will be disposed.